src/main/java/io/github/genomicdatainfrastructure/discovery/utils/PackageShowMapper.java

```diff
@@ -42,6 +42,7 @@ public RetrievedDataset from(CkanPackage ckanPackage) {
                 .url(ckanPackage.getUrl())
                 .languages(values(ckanPackage.getLanguage()))
                 .contact(value(ckanPackage.getContactUri()))
+                .creator(creators(ckanPackage))
```

Comment:
creator**s** is accept a list of creators.

Reply:
Done